ts/components/conversation/message/message-content/MessageQuote.tsx

```diff
@@ -27,15 +26,30 @@ export const MessageQuote = (props: Props) => {
   const multiSelectMode = useSelector(isMessageSelectionMode);
   const isMessageDetailViewMode = useSelector(isMessageDetailView);
 
+  if (!selected || isEmpty(selected)) {
+    return null;
+  }
+
   const quote = selected ? selected.quote : undefined;
-  const direction = selected ? selected.direction : undefined;
+  const direction = selected ? selected.direction : props.direction ? props.direction : undefined;
+
+  if (!quote || isEmpty(quote)) {
+    return null;
+  }
+
+  const quoteNotFound = Boolean(
+    !quote?.author || !quote.id || !quote.convoId || quote.referencedMessageNotFound
+  );
```

Comment:
that `quote.referencedMessageNotFound` is the most likely to happen so I'd put in first

`    quote.referencedMessageNotFound || quote.author || !quote.id || !quote.convoId
`

Reply:
Fair point